lib/GridItem.jsx

```diff
@@ -415,6 +458,106 @@ export default class GridItem extends React.Component<Props, State> {
     );
   }
 
+  delayEvents = React.createRef();
+  addEvent = (type, event, passive = true) => {
```

Comment:
Name could use some work; `addChildEvent` perhaps?

Reply:
Ok, I'll change it to addChildEvent